egs/swbd/s5c/local/chain/tuning/run_tdnn_blstm_1b.sh

```diff
@@ -0,0 +1,254 @@
+#!/bin/bash
+
+# tdnn_blstm_1b is same as tdnn_blstm_1b, but with the more tdnn layers
```

Comment:
do you mean "as tdnn_blstm_1a", here?  Is this comment correct?  I'm confused if the fast-lstm was the only change, as the num-params seems quite a but different.